.github/workflows/gradle-publish.yml

```diff
@@ -4,10 +4,6 @@
 name: Gradle Package
 
 on:
-  # TODO: Remove push trigger
-  push:
-    branches:
-      - main  
```

Comment:
Cleanup from my initial pass at packaging; forgot that I had not pushed this directly to master.